queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT  name, escape_attempts from animals WHERE   weight_kg > 10.5;
 SELECT * from animals WHERE   neutered = true;
 SELECT * from animals WHERE   name <> 'Gabumon';
 SELECT * from animals WHERE  weight_kg >= 10.4 and weight_kg  <= 17.3;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+UPDATE animals SET species = 'pokemon' WHERE species is null;
```

Comment:
- [ ] Please remove as this is a duplicate code.

Reply:
Definitely!